sdks/typescript/src/index.ts

```diff
@@ -1,92 +1,8 @@
-import {
+import { EtherealPulse } from './EtherealPulse';
```

Comment:
Minor thing but you can export the class directly, no need to import it

Reply:
done.